keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/ApplicationLoadBalancerHandlerTests.kt

```diff
@@ -65,7 +66,6 @@ internal class ApplicationLoadBalancerHandlerTests : JUnit5Minutests {
     repository,
     publisher
   )
-  private val mapper = ObjectMapper().registerKotlinModule()
```

Comment:
We weren't using `mapper` anywhere in this test so I assumed it was safe to delete.